lib/washout_builder.rb

```diff
@@ -44,7 +44,9 @@ def self.keys
 
     def self.config
       original_config.merge(description: nil)
+      original_config.merge(args_description: nil)
```

Comment:
i think here should be just one line with:
 ```ruby
 original_config.merge(description: nil, args_description: nil)
```
Otherwise you will override the first merge